core/src/main/kotlin/fr/sncf/osrd/api/project_signals/SignalProjectionRequest.kt

```diff
@@ -14,7 +14,7 @@ class SignalProjectionRequest(
     val blocks: List<String>,
     @Json(name = "track_section_ranges") var trackSectionRanges: List<DirectionalTrackRange>,
     var routes: List<String>,
-    @Json(name = "train_simulations") var trainSimulations: Map<Long, TrainSimulation>,
+    @Json(name = "train_simulations") var trainSimulations: Map<Int, TrainSimulation>,
```

Comment:
You mapped the wrong type (https://kotlinlang.org/docs/unsigned-integer-types.html).

```suggestion
    @Json(name = "train_simulations") var trainSimulations: Map<ULong, TrainSimulation>,
```